go/service/config.go

```diff
@@ -162,6 +162,24 @@ func (h ConfigHandler) GetFullStatus(ctx context.Context, sessionID int) (res *k
 	return status.GetFullStatus(mctx)
 }
 
+func (h ConfigHandler) IsServiceRunning(ctx context.Context, sessionID int) (res bool, err error) {
+	mctx := libkb.NewMetaContext(ctx, h.G()).WithLogTag("CFG")
+	defer mctx.TraceTimed("IsServiceRunning", func() error { return err })()
+
+	extStatus, err := status.GetExtendedStatus(mctx)
```

Comment:
Don't call this, this calls out to network too. Just figure out how it's computing `Service.Running` and copy that code

Reply:
oh rip okk